30-remote-site-setup.sh

```diff
@@ -153,7 +158,18 @@ fi
 echo "Using Bosco tarball: $(bosco_findplatform --url)"
 for ruser in $users; do
     setup_ssh_config
-    [[ $cvmfs_wn_client == 'no' ]] && setup_endpoints_ini
+done
+
+###################
+# REMOTE COMMANDS #
+###################
+
+# We have to pick a user for SSH, may as well be the first one
+remote_os_info=$(fetch_remote_os_info "$(printf "%s\n" $users | head -n1)" "$remote_fqdn")
+remote_os_ver=$(echo "$remote_os_info" | awk -F '=' '/^VERSION_ID/ {print $2}' | tr -d '"')
```

Comment:
> Also use /etc/os-release for better cross-platform support

Could you elaborate?

Reply:
/etc/redhat-release only exists on RHEL-derived distros; /etc/os-release works everywhere.

Reply:
> > I admit i'm real curious about what breaks with `rpm -E %rhel`
> Doesn't work on Ubuntu.

I'm guessing `[[ $remote_os_major_ver -gt 6 ]]` is also meaningless on ubuntu though; or at least misleading.

Reply:
Just misleading! We skip that function if it's a non-EL system